Tests/scripts/infrastructure_tests/release_notes_generator_test.py

```diff
@@ -592,3 +593,45 @@ def test_construct_entities_block_indicator_types(self):
         rn = construct_entities_block(entities_data)
         assert '### Indicator Types' in rn
         assert '- **accountRep**' in rn
+
+    @pytest.mark.parametrize('Pack_name, versions_ls, expected_results, expected_version', [
+        ("FakePack7", ["1_0_1.md", "1_0_2.md"],
+         "#### Integrations\n***Breaking Change*** some change\n\n##### entity1\n- Fixed something\n\n#### Scripts\n"
```

Comment:
Let's put the expected result in a dedicated file, such that instead of passing the results themselves you will be passing the filename.